packages/@react-aria/utils/src/scrollIntoView.ts

```diff
@@ -73,23 +73,18 @@ export function scrollIntoView(scrollView: HTMLElement, element: HTMLElement) {
  * offsetLeft or offsetTop through intervening offsetParents.
  */
 function relativeOffset(ancestor: HTMLElement, child: HTMLElement, axis: 'left'|'top') {
-  const prop = axis === 'left' ? 'offsetLeft' : 'offsetTop';
-  let sum = 0;
-  while (child.offsetParent) {
-    sum += child[prop];
-    if (child.offsetParent === ancestor) {
-      // Stop once we have found the ancestor we are interested in.
-      break;
-    } else if (child.offsetParent.contains(ancestor)) {
-      // If the ancestor is not `position:relative`, then we stop at
-      // _its_ offset parent, and we subtract off _its_ offset, so that
-      // we end up with the proper offset from child to ancestor.
-      sum -= ancestor[prop];
-      break;
-    }
-    child = child.offsetParent as HTMLElement;
-  }
-  return sum;
+  let childRect = child.getBoundingClientRect();
```

Comment:
We're a little nervous about this change. We think it's missing some cases. Looking at some other libraries, there is a fair amount of complexity in determining this. For example https://www.npmjs.com/package/compute-scroll-into-view

We might be able to use this library, though are reluctant to bring in the size increase. So the thing we need to prove is that this change is "enough" for our use cases or decide we're ok with the size increase.